### ISSUE TITLE
Allow server admins to configure who can use the /modviewer command

### ISSUE THREAD

**Issue by Elderweiss-was-taken** [haykam821/Mod-Viewer#3]
Either a whitelist/blacklist
Either changing the OP level and allowing to set it to zero so everyone can see the mods installed server side

**Comment by haykam821** [comment]
You can already use the `modviewer.command` permission key to control access on a finer level with a permissions mod that supports the Fabric Permissions API.

**Comment by Elderweiss-was-taken** [comment]
Yes but it's an entirely different mod, most mods just have a config file that you can edit in root/config, I don't see the security risk especially for a mod that only has read only features

I'm just gonna put the op-permission-level to zero and make everyone a OP on my server and boom problem solved

That or adding a config file that's not in a permission mod
